src/main/java/oss/fosslight/common/CoConstDef.java

```diff
@@ -818,23 +819,79 @@ public enum CAMPAIGN_ROUTE {
 	public static final String CD_DTL_LDAP_SEARCH_PW							= "200";
-	
+
 	/* SPDX CODE */
 	public static final String CD_SPDX											= "spdx";
 	public static final String CD_SPDX_RPF										= "spdxRdf";
 	public static final String CD_SPDX_TAG										= "spdxTag";
 	public static final String CD_SPDX_JSON										= "spdxJson";
 	public static final String CD_SPDX_YAML										= "spdxYaml";
-	
+
 	/* SCORE CODE */
 	public static final String CD_VULNERABILITY_MAILING_SCORE					= "750";
 	public static final String CD_VULNERABILITY_MAILING_SCORE_STANDARD			= "100";
-	
+
 	/* SECURITY SCORE CODE */
 	public static final String CD_SECURITY_VULNERABILITY_SCORE					= "760";
 	public static final String CD_SECURITY_VULNERABILITY_DETAIL_SCORE			= "100";
+
+	public enum HttpStatusCode {
+		HTTP_ACCEPTED(HttpURLConnection.HTTP_ACCEPTED),
+		HTTP_BAD_GATEWAY(HttpURLConnection.HTTP_BAD_GATEWAY),
+		HTTP_BAD_METHOD(HttpURLConnection.HTTP_BAD_METHOD),
+		HTTP_BAD_REQUEST(HttpURLConnection.HTTP_BAD_REQUEST),
+		HTTP_CLIENT_TIMEOUT(HttpURLConnection.HTTP_CLIENT_TIMEOUT),
+		HTTP_CONFLICT(HttpURLConnection.HTTP_CONFLICT),
+		HTTP_CREATED(HttpURLConnection.HTTP_CREATED),
+		HTTP_ENTITY_TOO_LARGE(HttpURLConnection.HTTP_ENTITY_TOO_LARGE),
+		HTTP_FORBIDDEN(HttpURLConnection.HTTP_FORBIDDEN),
+		HTTP_GATEWAY_TIMEOUT(HttpURLConnection.HTTP_GATEWAY_TIMEOUT),
+		HTTP_GONE(HttpURLConnection.HTTP_GONE),
+		HTTP_INTERNAL_ERROR(HttpURLConnection.HTTP_INTERNAL_ERROR),
+		HTTP_LENGTH_REQUIRED(HttpURLConnection.HTTP_LENGTH_REQUIRED),
+		HTTP_MOVED_PERM(HttpURLConnection.HTTP_MOVED_PERM),
+		HTTP_MOVED_TEMP(HttpURLConnection.HTTP_MOVED_TEMP),
+		HTTP_MULT_CHOICE(HttpURLConnection.HTTP_MULT_CHOICE),
+		HTTP_NO_CONTENT(HttpURLConnection.HTTP_NO_CONTENT),
+		HTTP_NOT_ACCEPTABLE(HttpURLConnection.HTTP_NOT_ACCEPTABLE),
+		HTTP_NOT_AUTHORITATIVE(HttpURLConnection.HTTP_NOT_AUTHORITATIVE),
+		HTTP_NOT_FOUND(HttpURLConnection.HTTP_NOT_FOUND),
+		HTTP_NOT_IMPLEMENTED(HttpURLConnection.HTTP_NOT_IMPLEMENTED),
+		HTTP_NOT_MODIFIED(HttpURLConnection.HTTP_NOT_MODIFIED),
+		HTTP_OK(HttpURLConnection.HTTP_OK),
+		HTTP_PARTIAL(HttpURLConnection.HTTP_PARTIAL),
+		HTTP_PAYMENT_REQUIRED(HttpURLConnection.HTTP_PAYMENT_REQUIRED),
+		HTTP_PRECON_FAILED(HttpURLConnection.HTTP_PRECON_FAILED),
+		HTTP_PROXY_AUTH(HttpURLConnection.HTTP_PROXY_AUTH),
+		HTTP_REQ_TOO_LONG(HttpURLConnection.HTTP_REQ_TOO_LONG),
+		HTTP_RESET(HttpURLConnection.HTTP_RESET),
+		HTTP_SEE_OTHER(HttpURLConnection.HTTP_SEE_OTHER),
+		HTTP_UNAUTHORIZED(HttpURLConnection.HTTP_UNAUTHORIZED),
+		HTTP_UNAVAILABLE(HttpURLConnection.HTTP_UNAVAILABLE),
+		HTTP_UNSUPPORTED_TYPE(HttpURLConnection.HTTP_UNSUPPORTED_TYPE),
+		HTTP_USE_PROXY(HttpURLConnection.HTTP_USE_PROXY),
+		HTTP_VERSION(HttpURLConnection.HTTP_VERSION);
+
+		private final int code;
+
+		HttpStatusCode(int code) {
+			this.code = code;
+		}
+
+		public int getCode() {
+			return code;
+		}
+		public static String httpCodePrint(int code) {

```

Comment:
In my opinion, the name of the method `httpCodePrint` could be more succinctly expressed, since the enum implies its meaning.